_pytest/runner.py

```diff
@@ -134,7 +135,11 @@ def _update_current_test_var(item, when):
     """
     var_name = 'PYTEST_CURRENT_TEST'
     if when:
-        os.environ[var_name] = '{0} ({1})'.format(item.nodeid, when)
+        value = _ascii_escaped('{0} ({1})'.format(item.nodeid, when))
```

Comment:
So does this `_ascii_escaped` call do anything here? I think it does nothing as we discovered, and if that's true, it should probably be removed to not cause further confusion.